pages/api/tool/deleteOne.ts

```diff
@@ -0,0 +1,15 @@
+import { Tool } from "database/models/tool";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const deleteOne = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const { id } = req.query;
+
+    await Tool.findByIdAndDelete(id)
+        .then((tool) => res.status(200).json(tool))
+        .catch((err) => res.status(500).send(err));
```

Comment:
will this throw an error if no tool for given id is found?

Reply:
this won't throw an error if no tool has that id - should we throw an error/return 404?